This patch prevents a warning message from being generated when the input
 file is not an xccdf results file.
This patch has not been submitted upstream, but is planned to be done by
 2015-Jan-31.

--- openscap-1.1.1/xsl/xccdf-guide.xsl.~1~	2014-10-29 10:01:23.785316068 -0700
+++ openscap-1.1.1/xsl/xccdf-guide.xsl	2014-10-29 09:55:47.880656520 -0700
@@ -81,4 +81,7 @@
     </xsl:call-template>
 </xsl:template>
 
+<!-- override warn-unresolved for guide generation -->
+<xsl:variable name='warn-unresolved' select='0' />
+
 </xsl:stylesheet>
